BP/common/culture/cultures/00_berber.txt

```diff
@@ -0,0 +1,1437 @@
+﻿berber_group = {
+	graphical_cultures = {
+		berber_group_building_gfx
+		berber_group_coa_gfx
+		mena_building_gfx
+		mena_clothing_gfx
+		mena_unit_gfx
+	}
+
+	mercenary_names = {
+		{ name = "mercenary_company_mamluks" coat_of_arms = "mc_mamluks" }
+	}
+	
+	#WtWSMS Berbers
+	berber = {
+		color = { 0.5 0.34 0.34 }
+		
+		cadet_dynasty_names = {
+			"dynn_Agerzam"
+			"dynn_Izil"
+		}
+		
+		dynasty_names = {
+			"dynn_Agerzam"
+			"dynn_Izil"
+		}
+		
+		male_names = {
+			Aderfu Afalkay Afaw Afer Amalu Amanar Amayyas Amazigh Amazzak Amamzu Ameqran Amesggin Amess Amestida Amezyan
+			Amnay Anaruz Areksim Aslal Atbir Awsim Aylal Azellay
+			Badis Bukkus
+			Gwafa Gwasila
+			Idder Idir Ides Igider Ikken Ilatig Ilayatmas Irate Isula Itri Izdarasen Izem Izemrasen Izil
+			Masinissa Massensen Massin Maysar Meddur Menna Merin Munatas
+			Saden Sifaks
+			Tacfin Takfarinas Tanana Tariq
+			Udad Ugwistan Usem Usus
+			Wimmiden Winaruz Winifsan Winsen Wiwurgh
+			Yani Yattuy Yuba Yufayyur Yufitran Yuften Yugerten
+			Ziri
+		}
+		
+		female_names = {
+			Basil
+			Damyan Dassin Dihya
+			Guraya
+			Hennu Herru
+			Ijja Illi Ittu
+			Kella Kweller
+			Lalla Lunja
+			Mamma Mammas Markunda Meghighda Menna
+			Rate
+			Siman
+			Tadefi Taderfit Tadla Tafalkayt Tafsut Tagafayt Taghbalut Tagwerramt Tagwilalt Tagwizult Thiyya Tajeddigt
+			Takensust Takama Taksimt Tala Tamalut Tamanart Tamazzalt Tamenzut Tameqrant Tanazart Tanebdatt Tanirt
+			Tanemghurt Tasafut Tatbirt Taweckint Tawenza Tawzalt Taylat Tayri Tazellayt Tidir Tinghan Tilelli Tinifsan
+			Tintziri Tuftifawt Tumert Tunaruz Tzeddig Tzila
+			Ultafa Ultasila
+		}
+		
+		dynasty_of_location_prefix = "dynnp_mn"
+		
+		# Chance of male children being named after their paternal or maternal grandfather, or their father. Sum must not exceed 100.
+		pat_grf_name_chance = 60
+		mat_grf_name_chance = 20
+		father_name_chance = 10
+		# Chance of female children being named after their paternal or maternal grandmother, or their mother. Sum must not exceed 100.
+		pat_grm_name_chance = 40
+		mat_grm_name_chance = 20
+		mother_name_chance = 30
+		
+		# Patronyms. Names after the primary parent. Can use both prefix and suffix together ("McDavidson"). _vowel is used for when the parent's name starts with a vowel.
+		patronym_prefix_male = "dynpat_pre_mmi"
+		patronym_prefix_female = "dynpat_pre_yeggi"
+		
+		always_use_patronym = no
+		
+		# The weight says how common the ethnicity is within the culture
+		ethnicities = {
+			10 = arab
+			1 = african
+		}
+		
+		dynasty_title_names = no
+		founder_named_dynasties = yes
+	}
+	guanches = {
+		graphical_cultures = {
```

Comment:
You don't need to include the entire graphical culture from above if you're only changing one thing.

Reply:
Does graphical cultures overwrite, or does it add to the culture groups graphics? I meant to add additional grpahics for guanches .

Reply:
It seems to overwrite. I say seems because the actual execution in-game seems mixed but in general, it'll take the graphical_culture listed first from the culture and if it can't find any for a category then from the culture group.

Reply:
Well, I meant for it to draw from all those graphical cultures, so I'll have to rethink the guanches then.

Reply:
>  I meant for it to draw from all those graphical cultures
That's what I said it does.

Reply:
Aight, I was confused by the overwrite comment. I can fix that soon.